bigbluebutton-html5/imports/api/external-videos/server/methods/startWatchingExternalVideo.js

```diff
@@ -1,32 +1,26 @@
 import { check } from 'meteor/check';
 import Logger from '/imports/startup/server/logger';
-import Users from '/imports/api/users';
 import RedisPubSub from '/imports/startup/server/redis';
 import { extractCredentials } from '/imports/api/common/server/helpers';
 
-export default function startWatchingExternalVideo(options) {
+export default function startWatchingExternalVideo(externalVideoUrl) {
   const REDIS_CONFIG = Meteor.settings.private.redis;
   const CHANNEL = REDIS_CONFIG.channels.toAkkaApps;
   const EVENT_NAME = 'StartExternalVideoPubMsg';
 
-  const { meetingId, requesterUserId: userId } = extractCredentials(this.userId);
-  const { externalVideoUrl } = options;
-
   try {
+    const { meetingId, requesterUserId } = extractCredentials(this.userId);
+
     check(meetingId, String);
-    check(userId, String);
+    check(requesterUserId, String);
     check(externalVideoUrl, String);
```

Comment:
We check here but not on `startExternalVideo`. Is this sufficient?

Reply:
We pushed those checks to the modifier but the `userId` is missing, indeed.